pkg/scanners/kubernetes/parser/manifest.go

```diff
@@ -0,0 +1,28 @@
+package parser
+
+import "gopkg.in/yaml.v3"
+
+type Manifest struct {
+	Path    string
+	Content ManifestNode
+}
+
+func (m *Manifest) UnmarshalYAML(value *yaml.Node) error {
+
+	switch value.Tag {
+	case "!!map":
+		node := make(ManifestNode)
+		if err := value.Decode(&node); err != nil {
+			return err
+		}
+		m.Content = node
+	default:
+		panic(value.Tag)
```

Comment:
Can we return an error here instead? Or is it game over if this is wrong?

Reply:
nope, its half finished with line numbers